packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -138,93 +139,64 @@ abstract class MaterialLocalizations {
   }
 }
 
-/// Localized strings for the material widgets.
+class _MaterialLocalizationsDelegate extends LocalizationsDelegate<MaterialLocalizations> {
+  const _MaterialLocalizationsDelegate();
+
+  @override
+  Future<MaterialLocalizations> load(Locale locale) => DefaultMaterialLocalizations.load(locale);
+
+  @override
+  bool shouldReload(_MaterialLocalizationsDelegate old) => false;
+}
+
+/// US English strings for the material widgets.
+///
+/// See also:
+///
+///  * [GlobalMaterialLocalizations], which provides material localizations for
+///    many languages.
 class DefaultMaterialLocalizations implements MaterialLocalizations {
   /// Constructs an object that defines the material widgets' localized strings
-  /// for the given `locale`.
+  /// for US English (only).
   ///
   /// [LocalizationsDelegate] implementations typically call the static [load]
   /// function, rather than constructing this class directly.
-  DefaultMaterialLocalizations(this.locale)
-      : this._localeName = _computeLocaleName(locale) {
-    assert(locale != null);
-    if (localizations.containsKey(locale.languageCode))
-      _nameToValue.addAll(localizations[locale.languageCode]);
-    if (localizations.containsKey(_localeName))
-      _nameToValue.addAll(localizations[_localeName]);
-
-    if (intl.NumberFormat.localeExists(_localeName)) {
-      _decimalFormat = new intl.NumberFormat.decimalPattern(_localeName);
-      _twoDigitZeroPaddedFormat = new intl.NumberFormat('00', _localeName);
-    } else if (intl.NumberFormat.localeExists(locale.languageCode)) {
-      _decimalFormat = new intl.NumberFormat.decimalPattern(locale.languageCode);
-      _twoDigitZeroPaddedFormat = new intl.NumberFormat('00', locale.languageCode);
-    } else {
-      _decimalFormat = new intl.NumberFormat.decimalPattern();
-      _twoDigitZeroPaddedFormat = new intl.NumberFormat('00');
-    }
+  DefaultMaterialLocalizations(Locale locale) {
+    _decimalFormat = new intl.NumberFormat.decimalPattern('en_US');
+    _twoDigitZeroPaddedFormat = new intl.NumberFormat('00', 'en_US');
   }
 
-  /// The locale for which the values of this class's localized resources
-  /// have been translated.
-  final Locale locale;
-
-  final String _localeName;
-
-  final Map<String, String> _nameToValue = <String, String>{};
-
-  /// Formats numbers using variable length format with no zero padding.
-  ///
-  /// See also [_twoDigitZeroPaddedFormat].
+  // Formats numbers using variable length format with no zero padding.
+  //
+  // See also [_twoDigitZeroPaddedFormat].
   intl.NumberFormat _decimalFormat;
 
-  /// Formats numbers as two-digits.
-  ///
-  /// If the number is less than 10, zero-pads it.
+  // Formats numbers as two-digits.
+  //
+  // If the number is less than 10, zero-pads it.
```

Comment:
We don't do this in the flutter source code as far as I know.  For example check out the many private methods and fields in the Element class.

Reply:
Oh, you did respond here. It's probably because people assume that `///` doesn't work (since they don't show up in dartdocs), and because it's a little known guideline in the style guide. However, it does work, e.g. in IDEs the analyzer can follow links, and the style guide recommends using it. We should update the rest of the code. Maybe I'll do that at some point.

Reply:
I've grown used to seeing // for internals fields and methods. We'd have to agree to update the entire codebase to ///.

Reply:
We should only use `///` for high-quality documentation. We use `//` for low-quality documentation (typical of the documentation we have for privates) because that way when we make it public we have to actually go and make sure the docs are good.

IDEs should allow you to follow links regardless of `///` vs `//`; if they don't file bugs.

Reply:
This is not in our style guide, and it is in Dart's normal style guide. If we care, we should document it in ours.

I do not understand the "high-quality" vs "low-quality" argument, and right now I don't have the cycles to think about it. Perhaps IDEs should follow links in `//`, but they don't, so not using `///` will make code navigation experience strictly worse. If we're willing to pay this cost, by all means let's use `//`.